app/views/stafftools/assignment_repos/_assignment_repo.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="flex-table classroom-resource">
+  <div class="flex-table-item">
+    <span class="assignment-icon assignment-icon-individual left">
+      <%= octicon('repo') %>
+    </span>
+
+    <div class="student-information">
+      <%= link_to stafftools_assignment_repo_path(assignment_repo.id) do %>
+        <strong class="css-truncate-target login">Assignment Repo: <%= assignment_repo.id %></strong>
+      <% end %>
+    </div>
+
```

Comment:
@johndbritton I think you're looking for the this page: https://github.com/education/classroom/blob/371c676f25264760c0fcf31bf48008eb2d62086b/app/views/stafftools/assignment_repos/show.html.erb

The `show` view should give all the details about it right?